.github/workflows/lgtm.yml

```diff
@@ -0,0 +1,15 @@
+name: Send LGTM reaction
+on:
+  issue_comment:
+    types: [created]
+  pull_request_review:
+    types: [submitted]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@1.0.0
+      - uses: micnncim/action-lgtm-reaction@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.SLIDE_WORKFLOWS_TOKEN }}
+          override: true
```

Comment:
@mahimachander maybe I acted to soon. If you would like, could you fix the override option here? It should not be in the env section, instead it should be in a `with` section, looking something like

```
with:
  overide: true
```